lib/src/validator/strict_dependencies.dart

```diff
@@ -96,10 +96,15 @@ class StrictDependenciesValidator extends Validator {
 
   /// Validates that no Dart files in `benchmark/`, `example/, `test/` or
   /// `tool/` have dependencies that aren't in [deps] or [devDeps].
+  ///
+  /// `example/` is only validated if it does not contain its own pubspec.
   void _validateBenchmarkExampleTestTool(
       Set<String> deps, Set<String> devDeps) {
-    for (var usage
-        in _usagesBeneath(['benchmark', 'example', 'test', 'tool'])) {
+    var directories = ['benchmark', 'test', 'tool'];
+    if (!fileExists(entrypoint.root.path('example', 'pubspec.yaml'))) {
```

Comment:
Hmm, it's it better to either care about the contents of the `example/` folder, or not care at all.

Reply:
Makes sense.
Done